tests/parser.mocha.spec.js

```diff
@@ -206,6 +212,26 @@ describe('Parser - Mocha Json', () => {
       ]
     });
   });
+  it('has multiple tags', () => {
+    const result = parse({ type: 'mocha', files: [`${testDataPath}/multiple-suites-multiple-tests-tags.json`] });
+    let testcase = result.suites[0].cases[0];
+    assert.equal(testcase.meta_data.has("tags"), true);
+    assert.equal(testcase.meta_data.get("tags"), "@fast,#1255")
```

Comment:
to make it consistent across all the test result formats, we can skip including the special symbols `@` and `#`.

Reply:
Agreed, stripping out the `@` and `#` makes sense for consistency sake. Though I do wonder if there's a scenario where teams need to differentiate between the two types of prefixes. Perhaps a good compromise might be to preserve the original tags as `tagsRaw`?:

```typescript
testcase.meta_data.set("tags", "fast,1234");
testcase.meta_data.set("tagsRaw", "@fast,#1234");
```